### ISSUE TITLE
Missing modules Math::Int64 and Math::Int128

### ISSUE THREAD

**Issue by pkdetlefsen** [maxmind/MaxMind-DB-Reader-XS#27]
`MaxMind::DB::Reader` started using `MaxMind::DB::Reader:PP` instead of the XS implementation in our CentOS 7 based Docker container yesterday. Here's how we set up the module in our Dockerfile:

```
RUN	curl -L https://github.com/maxmind/libmaxminddb/releases/download/1.3.2/libmaxminddb-1.3.2.tar.gz > /tmp/libmaxminddb-1.3.2.tar.gz && \
	cd /tmp && \
	tar -zxf libmaxminddb-1.3.2.tar.gz && \
	cd libmaxminddb-1.3.2 && \
	./configure && \
	make && \
	make install && \
	echo /usr/local/lib >> /etc/ld.so.conf.d/local.conf && \
	ldconfig && \
	cd .. && \
	rm -rf libmaxminddb-1.3.2*

RUN	cpanm -n -q MaxMind::DB::Reader::XS
```

When I tried loading it manually using `perl -MMaxMind::DB::Reader::XS` it complained that it couldn't locate `Math::Int64` and `Math::Int128`. After installing those two it worked again.

`cpanm -n -q Math::Int64 Math::Int128`

I suspect something changed might have changed in version 1.000006 or maybe there are some missing dependencies?

**Comment by mzealey** [comment]
This bit me too. It is very hard to see which module is being loaded, only when `use MaxMind::DB::Reader::XS` did I get these full errors.

**Comment by oschwald** [comment]
Thanks! This was fixed in v1.000007.
